### PR TITLE
Areas from cacti

### PR DESCRIPTION
**Pull request by fazerlicourice7**
Get areas from cacti.
Update arch search
separate analysis notebook for consistency
Start inverse pass area integration.

### CODE REVIEW

File: src/rcgen.py [hunk 1]
```diff
@@ -7,8 +7,8 @@ def generate_optimization_params(latency, active_power, active_energy, passive_p
         V_dd: voltage in V
         MemReadL: memory read latency in s
         MemWriteL: memory write latency in s
-        MemReadPact: memory read active power in W
-        MemWritePact: memory write active power in W
+        MemReadEact: memory read active power in W
```
Comment (patrickmcewen) [inline src/rcgen.py:10]: should say energy in J

File: src/params/tech_params.yaml [hunk 2]
```diff
@@ -366,28 +354,7 @@ latency: # ns
...
-  2097152: 13
-  4194304: 14
-  8388608: 15
-  16777216: 16
-  33554432: 17
-  67108864: 18
-  134217728: 19
-  268435456: 20
-  536870912: 21
+
```
Comment (patrickmcewen) [inline src/params/tech_params.yaml:357]: are we replacing latency/power scaling with something to account for different size-dependent costs?

**Review by patrickmcewen — COMMENTED** [review]
just a few small questions but looks good

**Comment by fazerlicourice7** [comment]
BIG ISSUE. Can no longer reproduce results from monday!! 
Debugging. - maybe one of the merges fucked up?

File: src/params/tech_params.yaml (hunk 2 above)
Comment (fazerlicourice7) [reply src/params/tech_params.yaml:357]: Cacti Integration handles this.

**Comment by fazerlicourice7** [comment]
stuff in params is all autogenerated, don't worry about this in the review

**Pull request merged**